cmd/hypper/lint_test.go

```diff
@@ -0,0 +1,40 @@
+// +build linux
```

Comment:
mmh, we should run the test on all OSes :/. TBF I don't see why the filepath walking would fail in Win, maybe we will be lucky.

Reply:
I think the output of the not found error is slightly different, so looking for the error text did not work, nor I have a easy available windows machine to test further :)

Reply:
Maybe it's possible to do what we did here:

https://github.com/rancher-sandbox/hypper/blob/441961614688aa0f338c2a22236d56cf6038f729/cmd/hypper/dependency_shared_test.go#L66-L69

I ended up trying a couple commits until I got the output correctly :P.

Reply:
oh god, the linting error is because of the windows paths vs linux paths....


dont do this to me.

Reply:
done!